### ISSUE TITLE
Converted jpeg to output instead of file

### ISSUE THREAD

**Issue by Kostanos** [monostream/tifig#18]
Hi, any way to output the jpeg to output buffer for use it in pipe?
Something like this:

    tifig -p img1.heic -o /dev/stdout | jpeg_handler_tool

**Comment by monostefan** [comment]
Hi @Kostanos, great idea! I made the output path parameter optional but omitting it will always convert to jpeg and ignore the `--verbose` parameter.  This allows you to do:

```
tifig img1.heic > converted.jpg
```

or in your scenario:

```
tifig -p img1.heic | jpeg_handler_tool
```

It's already released as version [0.2.1](https://github.com/monostream/tifig/releases/tag/0.2.1). Thanks for your help to improve tifig!

**Comment by monostefan** [comment]
PS: Feel free to reopen this issue if you detect any problem
